ibm/resource_ibm_private_dns_custom_resolver.go

```diff
@@ -169,13 +168,7 @@ func resouceIBMPrivateDNSCustomResolverCreate(context context.Context, d *schema
 	d.SetId(convertCisToTfTwoVar(*result.ID, crn))
 	d.Set(pdnsCRId, *result.ID)
 
-	_, err = waitForPDNSCustomResolverHealthy(d, meta)
```

Comment:
why is this wait logic removed?

Reply:
Wait logic added - It is triggered if location is provided